sdk/monitor/monitor-opentelemetry-exporter/samples/grpc/README.md

```diff
@@ -0,0 +1,71 @@
+# Overview
+
+OpenTelemetry gRPC Instrumentation allows the user to automatically collect trace data and export them to the backend of choice (we can use Zipkin or Jaeger for this example), to give observability to distributed systems.
+
+## Installation
+
+```sh
+# from this directory
+npm install
+```
+
+Setup [Zipkin Tracing](https://zipkin.io/pages/quickstart.html)
```

Comment:
any reason the link here doesn't include `Setup` as well?

Reply:
Removed zipkin/jaeger setup docs altogether